ginga/rv/Control.py

```diff
@@ -587,6 +587,12 @@ def load_image(self, filepath, idx=None, show_error=True):
         self.logger.debug("Successfully loaded file into object.")
         return image
 
+    def _dl_indicator(self, count, blksize, totalsize):
```

Comment:
@ejeschke , there is no reason why this cannot be a proper class method, right? It was previously buried in another method below.